packages/stryker-webpack-transpiler/src/fs/OutputFileSystem.ts

```diff
@@ -0,0 +1,94 @@
+import { PathLike } from 'fs';
+import * as path from 'path';
+import { webpack, EmptyCallback, Callback } from '../types';
+import { BinaryFile, TextFile, FileKind } from 'stryker-api/core';
+import ChunkSorter, { Chunk } from '../compiler/ChunkSorter';
+
+const binaryFileExtensions = Object.freeze(['.ico', '.zip']);
+
+export default class OutputFileSystem implements webpack.OutputFileSystem {
+
+  private _files: {
+    [name: string]: string | Buffer;
+  };
+
+  constructor() {
+    this.purge();
```

Comment:
I get why you call `purge` here (to initialize `this_files`) but it's weird to see a purge in a constructor.

Reply:
It's more DRY this way. It's not that strange right? You start out with a purged file system.